linchpin/provision/roles/aws/molecule/shared/prepare.yml

```diff
@@ -0,0 +1,15 @@
+- name: register RHSM
+  hosts: all
+  roles:
+    - role: oasis_roles.rhsm
+      when: ansible_distribution == 'RedHat'
+  vars:
+    rhsm_username: "{{ lookup('env', 'OASIS_RHSM_USERNAME') }}"
+    rhsm_password: "{{ lookup('env', 'OASIS_RHSM_PASSWORD') }}"
+    rhsm_server_hostname: "{{ lookup('env', 'OASIS_RHSM_SERVER_HOSTNAME') }}"
+    rhsm_pool_ids: "{{ lookup('env', 'OASIS_RHSM_POOL_IDS') }}"
+    rhsm_repositories:
+      enable:
+        - rhel-7-server-rpms
+        - rhel-7-server-extras-rpms
+        - rhsl-7-server-optional-rpms
```

Comment:
Why do we need RHSM?

Reply:
This is  a part of the OASIS defaults.  I don't quite know what its used for.  I can try deleting it and seeing what happens since the docker testing provider uses centos 7 instead of RHEL

Reply:
The short answer is that we don't right now.  However, I do have a couple of PRs coming up to add RHEL 7 and RHEL 8 support.  I can remove the lines referencing the rhel-related roles for oasis and then add them back in those PRs if you'd prefer.  Or I can just leave them in here to avoid breaking anything

Reply:
I'm a coding puritan but it doen't hurt and it has already an enabling condition.